pkgs/development/python-modules/aioitertools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, typing-extensions
+, coverage
+, toml
+, mypy
+}:
+
+buildPythonPackage rec {
+  pname = "aioitertools";
+  version = "0.7.1";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "18ql6k2j1839jf2rmmmm29v6fb7mr59l75z8nlf0sadmydy6r9al";
+  };
+
+  propagatedBuildInputs = [ typing-extensions ];
+  checkInputs = [ coverage toml mypy ];
+
+  checkPhase = ''
+    make test
```

Comment:
We only want to execute this command https://github.com/omnilib/aioitertools/blob/main/makefile#L30.

Reply:
done